pybind11-dev/install.yaml

```diff
@@ -0,0 +1,5 @@
+- type: system
+  default:
+    name: pybind11-dev
```

Comment:
A pip target maybe better as pip always has the latest pybind11. We can restrict it to a particular major release if needed.

Reply:
I just follow rosdistro

Reply:
Okay. That can be setup as a separate target as this package doesn't install the python modules.

Python module provides a convenience function called `pybind11.get_include()` which provides the location of the include directory and is useful in building extensions using `setup.py`